test_bot/test_external_moves.py

```diff
@@ -47,6 +47,30 @@ def online_book_get() -> OnlineType:
 
         return online_book_get()
 
+    def is_chessdb_cn_up(self) -> bool:
```

Comment:
This and the other new functions could be generalized into a `is_site_up(self, url)` function, and then used by `def is_chessdb_cn_up(self): return self.is_site_up("https://www.chessdb.cn/cdb.php")`.